hms-app/src/actions/HackathonActions.tsx

```diff
@@ -1,33 +1,27 @@
-const api_id = process.env.REACT_APP_API_ID
+const core_url = process.env.REACT_APP_CORE_URL
 
 export const getListOfHackathons = () => {
```

Comment:
so I think what could really make your code a bit simpler is to set the return type of these.  To make it work you would need to do something along the lines of....
- Change the signature to return a `Promise<Hackathon[]>` or something like that
- Remove the `catch` statement, this line makes it also possible for the function to return `Promise<void>`
  - You could also keep the line and rethrow the Error that you catch after you log it.  Either way if you keep the line and rethrow or remove it, the callers of this function need to handle the error somehow.
  - Another solution would be leave the line and change the signature to something like `Promise<void | Hackathon[]>` but then you need to be sure you handle the case in the frontend when this returns void in the things hat use this function
- Add another `.then()` line that converts the json to a Hackathon[].  This will add a good bit of code of course but will remove duplication from the callers of this method